param.go

```diff
@@ -287,13 +287,30 @@ func (po paramObject) DotParam() []*dot.Param {
 func newParamObject(t reflect.Type) (paramObject, error) {
 	po := paramObject{Type: t}
 
+	// Check if the In type supports ignoring unexported fields.
+	var ignoreUnexported bool
+	for i := 0; i < t.NumField(); i++ {
+		f := t.Field(i)
+		if f.Type == _inType {
+			var err error
+			ignoreUnexported, err = isIgnoreUnexportedSet(f)
```

Comment:
optional: should we invert the if condition?

```
if f.Type != _inType { continue }

var err error
// ...
```

Reply:
Hmm, we could. I went with `break` because we can stop processing the fields as soon as we find the `_inType` field . With `continue` we would continue looking through all fields.